superdesk/places/places_autocomplete.py

```diff
@@ -37,7 +37,7 @@ def get(self, req, lookup):
         assert req.args.get('name'), {'name': 1}
         params = [
             ('name_startsWith', req.args.get('name')),
-            ('lang', req.args.get('lang')),
+            ('lang', 'fr' if req.args.get('lang') == 'fr-CA' else req.args.get('lang')),
```

Comment:
imo would be good to use something generic like `lang.split('-')[0]` so it's not just french